lib/workers/pr/changelog/release-notes.ts

```diff
@@ -3,42 +3,71 @@ import { linkify } from 'linkify-markdown';
 import MarkdownIt from 'markdown-it';
 
 import { api } from '../../../platform/github/gh-got-wrapper';
+// import { api as api_gitlab } from '../../../platform/gitlab/gl-got-wrapper';
 import { logger } from '../../../logger';
 import { ChangeLogResult, ChangeLogNotes } from './common';
 
 const { get: ghGot } = api;
+// const { get: glGot } = api_gitlab;
 
 const markdown = new MarkdownIt('zero');
 markdown.enable(['heading', 'lheading']);
 
 export async function getReleaseList(
-  githubApiBaseURL: string,
+  apiBaseURL: string,
   repository: string
 ): Promise<ChangeLogNotes[]> {
   logger.trace('getReleaseList()');
   // istanbul ignore if
-  if (!githubApiBaseURL) {
+  if (!apiBaseURL) {
+    logger.debug('No apiBaseURL');
     return [];
   }
```

Comment:
We should have a test case for this. 😉

Reply:
We don't know which case this covers, that part of code was written before our changes.